src/Management/Commands/MuteCommand.cs

```diff
@@ -0,0 +1,72 @@
+﻿using sodoffmmo.Attributes;
+using sodoffmmo.Core;
+
+namespace sodoffmmo.Management.Commands;
+
+[ManagementCommand("mute", Role.Moderator)]
+class MuteCommand : IManagementCommand {
+    public void Handle(Client client, string[] arguments) {
+        if (arguments.Length < 1) {
+            client.Send(Utils.BuildServerSideMessage("Mute: Invalid number of arguments", "Server"));
+            return;
+        }
+        string name = string.Join(' ', arguments);
+        Client? target = client.Room.Clients.FirstOrDefault(x => x.PlayerData.DiplayName == name); // Find in this room.
+        if (target == null) { // Search all clients in case the offender left the room.
+            target = Client.GetAllClients().FirstOrDefault(x => x.PlayerData.DiplayName == name);
+            if (target == null) { // Failure
+                client.Send(Utils.BuildServerSideMessage($"Mute: user {name} not found", "Server"));
+                return;
+            }
+        }
+
+        if (target.PlayerData.VikingId != null) {
```

Comment:
This could be done the same way as ban/unban. Let target handle it, it makes a nicer api and we can use it elsewhere if we need it. `target.Mute()`